modules/ROOT/pages/proc_installing_cli.adoc

```diff
@@ -0,0 +1,50 @@
+[id='installing-cli_{context}']
+= Installing the CLI
+
+In order to interact with Tekton from a terminal, you need to install the Tekton CLI (`tkn`).
+
+== Installing from binary
+
+=== For macOS X
+
+.Get the `tar.xz`:
+----
+curl -LO https://github.com/tektoncd/cli/releases/download/v0.2.0/tkn_0.2.0_Darwin_x86_64.tar.gz
+----
+
+.Extract `tkn` to your PATH:
+----
+sudo tar xvzf tkn_0.2.0_Darwin_x86_64.tar.gz -C /usr/local/bin tkn
+----
+
+=== For Linux AMD64
+
+.Get the `tar.xz`:
+----
+curl -LO https://github.com/tektoncd/cli/releases/download/v0.2.0/tkn_0.2.0_Linux_x86_64.tar.gz
```

Comment:
The CLI versions should get updated to the 0.2.2 release

Reply:
Udated